src/healthcareapis/azext_healthcareapis/generated/custom.py

```diff
@@ -42,22 +42,32 @@ def healthcareapis_service_create(client,
                                   public_network_access=None,
                                   export_configuration_storage_account_name=None,
                                   no_wait=False):
+
+    properties = {
+        'access_policies': access_policies,
+        'cosmos_db_configuration': cosmos_db_configuration,
+        'authentication_configuration': authentication_configuration,
+        'cors_configuration': cors_configuration,
+        'private_endpoint_connections': private_endpoint_connections,
+        'public_network_access': public_network_access,
+        'export_configuration_storage_account_name': export_configuration_storage_account_name
+    }
+
+    service_description = {
+        'name': resource_name,
+        'kind': kind,
+        'location': location,
+        'tags': tags,
+        'etag': etag,
+        'identity_type': identity_type,
+        'properties': properties
+    }
+
     return sdk_no_wait(no_wait,
-                       client.begin_create_or_update,
+                       client.create_or_update,
```

Comment:
The change indicates that the SDK is reverted from track 2 to track 1, is this intentional?

Reply:
Can you explain the differences between track 2 and track 1? these changes were intentional because the previous code (track 2, I am assuming) weren't functioning properly when making calls, there were many missing property errors that were there from prior releases.

Also I am currently making changes in the PR found here https://github.com/Azure/azure-cli-extensions/pull/2749. We are trying to include all the changes in the above PR.